src/_utils/item/Item.tsx

```diff
@@ -108,12 +110,14 @@ const Item = ({
       <div className="kirk-item-leftText">
         {leftTitle && (
           <Text
-            className={leftBody ? 'kirk-item-title' : null}
```

Comment:
I think it's a legacy rule. There is no logic display link between leftBody inside leftTitle.
And kirk-item-title doesn't apply any styles.